vms/platformvm/service_test.go

```diff
@@ -623,7 +623,7 @@ func TestGetCurrentValidators(t *testing.T) {
 		found := false
 		for i := 0; i < len(response.Validators); i++ {
 			gotVdr := response.Validators[i].(pchainapi.PermissionlessValidator)
-			if gotVdr.NodeID != nodeID {
+			if gotVdr.NodeID.Compare(nodeID) != 0 {
```

Comment:
Why change from using `!=` to `.Compare` (here and below)?

Reply:
given that the `NodeID` is now a struct, using a comparison method allow us to ensure we can retain the comparison functionality even if we were to add, let's say, a non-comparable variable to the struct ( i.e. pointer ).
I figured that since we already having a comparison method, we should use it instead of relying on a mix of comparison techniques.